lib/smooth/include/smooth/application/display/DisplaySpi.h

```diff
@@ -166,14 +165,14 @@ namespace smooth::application::display
 
             smooth::core::io::Output dc_pin;
             smooth::core::io::Output cs_pin;
+            std::unique_ptr<DisplayPin> backlight_pin{ nullptr };
+            std::unique_ptr<DisplayPin> reset_pin{ nullptr };
```

Comment:
Zero-initialize: `backlight{}`, no need to specify nullptr.

Reply:
Ok will remove nullptr to both